mcu/atmega-hal/src/usart.rs

```diff
@@ -31,3 +31,64 @@ avr_hal_generic::impl_usart_traditional! {
     rx: port::PD2,
     tx: port::PD3,
 }
+
+#[cfg(feature = "atmega2560")]
+pub type Usart0<CLOCK, IMODE> = Usart<
+    crate::RawPeripheral<crate::pac::USART0>,
+    port::Pin<port::mode::Input<IMODE>, port::PE0>,
+    port::Pin<port::mode::Output, port::PE1>,
+    CLOCK,
+>;
+#[cfg(feature = "atmega2560")]
+avr_hal_generic::impl_usart_traditional! {
+    peripheral: crate::RawPeripheral<crate::pac::USART0>,
+    register_suffix: 0,
+    rx: port::PE0,
+    tx: port::PE1,
+}
+
+#[cfg(feature = "atmega2560")]
+pub type Usart1<CLOCK, IMODE> = Usart<
+    crate::RawPeripheral<crate::pac::USART1>,
+    port::Pin<port::mode::Input<IMODE>, port::PD2>,
+    port::Pin<port::mode::Output, port::PD3>,
+    CLOCK,
+>;
+#[cfg(feature = "atmega2560")]
+avr_hal_generic::impl_usart_traditional! {
+    peripheral: crate::RawPeripheral<crate::pac::USART1>,
+    register_suffix: 1,
+    rx: port::PD2,
+    tx: port::PD3,
+}
```

Comment:
This one looks exactly the same as the one for `ATmega23U4` so I think it makes sense to collapse them into a single implementation.

Reply:
fixed